packages/bot-web-ui/src/components/quick-strategy/inputs/qs-input.tsx

```diff
@@ -23,6 +26,38 @@ const QSInput: React.FC<TQSInput> = observer(
         const { setFieldValue, setFieldTouched } = useFormikContext();
         const is_number = type === 'number';
 
+        const sendPlusValueToRudderstack = (key: string, value: string) => {
+            Analytics.trackEvent('ce_bot_quick_strategy_form', {
+                action: 'change_parameter_value',
+                parameter_value: key,
+                plus_push_amount: value,
+            });
+        };
+
+        const sendMinusValueToRudderstack = (key: string, value: string) => {
+            Analytics.trackEvent('ce_bot_quick_strategy_form', {
+                action: 'change_parameter_value',
+                parameter_value: key,
+                minus_push_amount: value,
+            });
+        };
+
+        const throttleChangePlus = React.useCallback(
+            debounce(sendMinusValueToRudderstack, DEBOUNCE_INTERVAL_TIME, {
+                trailing: true,
+                leading: false,
+            }),
+            []
+        );
+
+        const throttleChangeMinus = React.useCallback(
+            debounce(sendPlusValueToRudderstack, DEBOUNCE_INTERVAL_TIME, {
+                trailing: true,
+                leading: false,
+            }),
+            []
+        );
+
```

Comment:
I saw in some places debounce effect looks good, but why we need this behaviour here? Why can't we send it immidiately?